cves/CVE-2017-5638.yml

```diff
@@ -45,7 +45,12 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  Apache Struts uses the Jakarta Multiparser, and it generates an exception when parsing 
+  the Content-Type. It raises this exception and includes the invalid data in the 
+  error message, and does this by parsing the input and treating the parsed result as an 
+  OGNL expression. This is potential for remote code execution. This affects Apache Struts 
+  2.3.5 - 2.3.31 and 2.5 - 2.5.10
```

Comment:
I think it'd be best if you removed any references to version numbers. If we wanted to have more detail, we could just look it up later.

Reply:
noted